dataactvalidator/scripts/initialize.py

```diff
@@ -62,7 +61,6 @@ def loadValidator():
 
 def setupDB():
     setupValidationDB()
-    setupStagingDB()
```

Comment:
It is called in setupAllDB, I had been under the impression that was called elsewhere.  Looking at the broker initialize, it seems that lists the DBs to be set up individually, so I've re-added the staging call here.